wagtail/contrib/redirects/templates/wagtailredirects/index.html

```diff
@@ -26,41 +26,18 @@
         {% url "wagtailredirects:report" as report_link %}
         {% trans "Import redirects" as import_str %}
         {% trans "Export redirects" as export_str %}
+        {% fragment as redirect_actions %}
+            <div class="dropdown dropdown-button match-width">
+                <a href="{{ import_link }}" class="button bicolor button--icon">{% icon name="doc-full-inverse" wrapped=1 %}{{ import_str }}</a>
+                <div class="dropdown-toggle">{% icon name="arrow-down" %}</div>
+                <ul>
+                    <li><a class="button bicolor button--icon" href="{{ report_link }}">{% icon name="download" wrapped=1 %}{{ export_str }}</a></li>
+                </ul>
+            </div>
+        {% endfragment %}
+        {% include "wagtailadmin/shared/header.html" with title=redirects_str icon="redirect" merged=1 search_url="wagtailredirects:index" query_parameters=query_parameters action_url=add_link action_text=add_str action_icon="plus" extra_actions=redirect_actions %}
```

Comment:
We don’t need the merged variant here.

```suggestion
        {% include "wagtailadmin/shared/header.html" with title=redirects_str icon="redirect" search_url="wagtailredirects:index" query_parameters=query_parameters action_url=add_link action_text=add_str action_icon="plus" extra_actions=redirect_actions %}
```

I can’t see anything named `query_parameters` in this template’s context, but I can also see you’ve just taken this from the existing code, so let’s leave this be.